### PR TITLE
info about migration

### PR DESCRIPTION
**Pull request by jburel**
Add info where the files are now
cc @pwalczysko

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,12 @@
 A set of scripts to access, analyse image and metadata from OMERO.
 
 Under ``practical`` are the files that can be used during a training session.
+The scripts under ``practical`` are no longer maintained.
```
Comment (pwalczysko) [inline README.md:5]:
```suggestion
Nevertheless, the ``scripts`` under ``practical`` are no longer maintained.
```

File: README.md [hunk 2]
```diff
@@ -2,4 +2,12 @@
...
+They have been moved to various repositories:
```
Comment (pwalczysko) [inline README.md:6]:
```suggestion
They have been moved to following repositories:
```

File: README.md [hunk 3]
```diff
@@ -2,4 +2,12 @@
...
+ - The files under ``practical/groovy`` have been moved to [omero-guide-fiji](https://github.com/ome/omero-guide-fiji/tree/master/scripts).
+ - The files under ``practical/javascript`` have been moved to [omero-guide-figure](https://github.com/ome/omero-guide-figure/tree/master/scripts).
+ - The files under ``practical/matlab`` have been moved to [omero-guide-matlab](https://github.com/ome/omero-guide-matlab/tree/master/scripts).
+ - The files under ``practical/orbit`` have been moved to [omero-guide-orbit](https://github.com/ome/omero-guide-orbit/tree/master/scripts).
+  - The files under ``practical/python`` have been moved to [omero-guide-python](https://github.com/ome/omero-guide-python/tree/master/scripts).
```
Comment (pwalczysko) [inline README.md:11]:
```suggestion
 - The files under ``practical/python`` have been moved to [omero-guide-python](https://github.com/ome/omero-guide-python/tree/master/scripts).
```

Not sure how many spaces should be from the left, but the last line is shifted here - although it still builds well in display

File: README.md [hunk 4]
```diff
@@ -2,4 +2,12 @@
 A set of scripts to access, analyse image and metadata from OMERO.
 
 Under ``practical`` are the files that can be used during a training session.
+Nevertheless, the ``scripts`` under ``practical`` are no longer maintained.
+They have been moved to following repositories:
```
Comment (pwalczysko) [inline README.md:6]:
```suggestion
The majority of the ``practical`` scripts have been moved to following repositories.
```

**Comment by pwalczysko** [comment]
One small cosmetic comment in https://github.com/ome/training-scripts/pull/87#discussion_r533344083 
One bigger comment about what is left (unmaintained) and what was moved somewhere else (i.e. maintained somewhwere else, in other repos) is https://github.com/ome/training-scripts/pull/87/files#r533350990

Otherwise looks fine.

**Comment by jburel** [comment]
I still have some files to move e.g. Jython scripts. This is the current status of the repository.

**Comment by pwalczysko** [comment]
> I still have some files to move e.g. Jython scripts. This is the current status of the repository.

I think this is fine. If we try to explain it in meticulous detail, we will just confuse ourselves and the statement would quickly become outdated anyway as you go ahead with the migration. The text as it is now in this PR is making fair understandable points.

**Review by pwalczysko — APPROVED** [review]

**Comment by pwalczysko** [comment]
once the checks have completed, ready to merge

**Comment by jburel** [comment]
Travis is hanging as often now.

**Pull request merged**